Casks/s/spectrolite.rb

```diff
@@ -0,0 +1,22 @@
+cask "spectrolite" do
+  arch arm: "-silicon", intel: ""
+
+  version "0.22.1"
+  sha256 intel: "465b4abb9413a53050c20e2adb4f95ffb9747b6927c5df35d3f5d6e1c7f633b2",
+         arm:   "42840adc89c91f371426588f20d8789c1769605f293180d8af0fd13cad929629"
+
+  url "https://spectrolite.app/downloads/Spectrolite#{arch}.dmg"
+  name "spectrolite"
+  desc "Helps make risograph prints more easily"
+  homepage "https://spectrolite.app/"
+
+  livecheck do
+    skip "Version information not reliably available"
+  end
+
```

Comment:
In app updater with yml file
```suggestion
  livecheck do
    url "https://d398dq9v5f85fz.cloudfront.net/latest-mac.yml"
    strategy :electron_builder
  end

  auto_updates true
```